tests/pipelines/test_pipelines_text_generation.py

```diff
@@ -435,8 +435,6 @@ def run_pipeline_test(self, text_generator, _):
                 ],
             )
 
-        with self.assertRaises(ValueError):
-            outputs = text_generator("test", return_full_text=True, return_text=True)
```

Comment:
I'm also not sure why we remove a check that they're mutually exclusive -- I think it's far better to raise an exception if both are set rather than have one silently override the other

Reply:
I was trying to reconcile the docs (which used to be very wrong) with the options, but you're right - I'll make them mutually exclusive like they were before, and update the docs instead.